presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -882,6 +887,16 @@ private void initializeOutputPositions(int positionCount)
         }
     }
 
+    private Block createRowNumbersBlock(int[] positions, int positionCount,
```

Comment:
can this be a static method ? also please name the parameter positionsToRead

Reply:
Sure